### PR TITLE
Ski Resort Area Locations

### PR DESCRIPTION
**Pull request by miriamseely**
Current metadata draft for [Utah Ski Resort Area Locations](https://opendata.gis.utah.gov/datasets/utah::utah-ski-area-resort-locations/about). Let me know what you think!

### CODE REVIEW

File: metadata/recreation/ski_area_resort_locations.md [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+
+## Summary
+
+Point locations for each of Utah's alpine ski areas and resorts. This dataset includes named point locations only and does not include pricing, resort conditions, or other skiing information.
+
+## Description
+
+### What is the dataset?
+
+This dataset contains point locations for alpine ski areas and resorts in Utah as of 2017. The resorts included in this dataset are Alta, Beaver Mountain, Brian Head, Brighton, Cherry Peak, Deer Valley, Eagle Point, Nordic Valley, Park City, Powder Mountain, Snowbasin, Snowbird, Solitude, and  Sundance.
```
Comment (steveoh) [inline metadata/recreation/ski_area_resort_locations.md:21]: remove extra space

```suggestion
This dataset contains point locations for alpine ski areas and resorts in Utah as of 2017. The resorts included in this dataset are Alta, Beaver Mountain, Brian Head, Brighton, Cherry Peak, Deer Valley, Eagle Point, Nordic Valley, Park City, Powder Mountain, Snowbasin, Snowbird, Solitude, and Sundance.
```

**Review by steveoh — APPROVED** [review]
I'm not sure if anyone would care or if I would suggest any changes but "alpine ski areas" could be interpreted in an exclusionary way for the mode of travel. Alpine in the sense of high mountains makes sense but it could also imply alpine skiing only where other modes of travel are acceptable. For instance snowboarding, telemark skiing, etc.

**Comment by miriamseely** [comment]
> I'm not sure if anyone would care or if I would suggest any changes but "alpine ski areas" could be interpreted in an exclusionary way for the mode of travel. Alpine in the sense of high mountains makes sense but it could also imply alpine skiing only where other modes of travel are acceptable. For instance snowboarding, telemark skiing, etc.

Ah, see I don't ski so I didn't realize that alpine skiing was referring to the type of skiing, not the geography. I'll take it out, I agree it sounds exclusionary. Thanks for your input!

File: metadata/recreation/ski_area_resort_locations.md [hunk 2]
```diff
@@ -30,11 +30,11 @@ Each point in this dataset has been placed on or near the main parking lot of th
 
 ### How was the dataset created?
 
-UGRC created this dataset by digitizing points on top of high resolution aerial photography. The data are maintained by UGRC and are updated as neeeded.
+UGRC created this dataset by digitizing points on top of high resolution aerial photography. The data are maintained by UGRC and are updated as needed.
```
Comment (ZachBeck) [inline metadata/recreation/ski_area_resort_locations.md:33]: Points were generally placed on or near the main parking area of the resort.

**Pull request merged**